services/src/main/java/org/keycloak/keys/JavaKeystoreKeyProvider.java

```diff
@@ -93,7 +93,8 @@ protected KeyWrapper loadKey(RealmModel realm, ComponentModel model) {
             String keyAlias = model.get(JavaKeystoreKeyProviderFactory.KEY_ALIAS_KEY);
 
             return switch (algorithm) {
-                case Algorithm.PS256, Algorithm.PS384, Algorithm.PS512, Algorithm.RS256, Algorithm.RS384, Algorithm.RS512 ->
+                case Algorithm.PS256, Algorithm.PS384, Algorithm.PS512, Algorithm.RS256, Algorithm.RS384, Algorithm.RS512,
+                        Algorithm.RSA_OAEP, Algorithm.RSA1_5, Algorithm.RSA_OAEP_256 ->
```

Comment:
I think that you need to add the `RSA_OAEP` to the property `algorithm`  in the factory ([here](https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/keys/JavaKeystoreKeyProviderFactory.java#L120)). Right? (If not I think you cannot configure it via admin console.)

Reply:
Thanks for the hint @rmartinc. Fixed with [3f33dcf](https://github.com/keycloak/keycloak/pull/29853/commits/3f33dcfee1a6c6dc262c0bcf869362af371ca516).